### DISCUSSION TITLE
No Website after installation

### DISCUSSION THREAD

**Discussion by srype** [magicbug/Cloudlog#1873] [Q&A]
Cloudlog is installed so far but I've got an issue with opening the install wizard. ![image](https://user-images.githubusercontent.com/101605900/209484772-b4a02743-14ef-4f8f-ab94-075bd8ed6c56.jpeg) This is the webpage that is displayed when I enter the local ip of the raspy. I'm not sure but I think something is wrong with the directory. Any help?

**Comment by phl0** [comment]
This is the default page of the Apache webserver. Where did you put the cloudlog files? Maybe just a subdirectory in /var/www/html? Then you would have to add that to the URL. E.g. http://192.168.x.x/cloudlog

**Reply by srype** [reply]
I'll take a look at the files. Maybe something has been moved to the wrong files. Which file system does DietPi use? ext4?

**Reply by phl0** [reply]
The file system does not cause issues here. What is the absolute path you extracted or moved the Cloudlog files to?

**Reply by srype** [reply]
I know that the Problem is not caused by the Filesystem. I work with a Windows Computer atm. Windows does not support Linux Filesystems. So I need to know wich filesystem DietPi uses to acces the Files via a separate Program on the SD-Card of the Raspy. The plan is to set up a FTP-Server for access via network. But so far I have not been able to get this to work.

**Reply by phl0** [reply]
I would recommend logging in via SSH and check on the system itself.

**Reply by srype** [reply]
I access the Pi via SSH already. But I have no clue about Linux-Commands.

**Reply by phl0** [reply]
Ok this is going to get a little hard then....
Into which dir did you extract the cloudlog sources?

**Reply by AndreasK79** [reply]
You could use WinSCP to connect. You can then see the file structure and the files. That way you do not need to know Linux commands.

**Reply by srype** [reply]
I extracted Cloudlog into /var/www/html.

**Reply by phl0** [reply]
Ok, so there is a directory cloudlog inside /var/www/html?

**Reply by srype** [reply]
> You could use WinSCP to connect. You can then see the file structure and the files. That way you do not need to know Linux commands. 

But how? With an FTP-Server on DietPi?

**Reply by phl0** [reply]
> > You could use WinSCP to connect. You can then see the file structure and the files. That way you do not need to know Linux commands.
> 
> But how? With an FTP-Server on DietPi?

Nope. WinSCP uses SSH protocol. So you can login using your default user.
